### DISCUSSION TITLE
suggestion: Add a toggle switch for `Hide double tap overlay filter`

### DISCUSSION THREAD

**Discussion by 98xe** [anddea/revanced-patches#217] [Ideas]
### Application

YouTube

### Suggestion description

The patch is included by default, and there's no way to turn it off afterwards. It would make sense to have it be a toggleable feature, similar to `Hide player buttons background`.

Maybe exclude it by default, too.

### Acknowledgements

- [X] This issue is not a duplicate of an existing suggestion.
- [X] This is not a feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by anddea** [comment]
Some of the options do not have an obvious implementation, and such things are better to be left off as they are, otherwise you might face some performance issues. I'm going to close it now, I'll think about possible implementations without giving up on performance. 

This option is turned on by default because it is less laggy and also less heavy on performance because it doesn't need to fade back and forth, especially if you double tap a lot. And second reason, you won't miss out the content. I'm trying to enable only those options and patches that will actually improve performance, UI, etc. And (if you want to) you can exclude, disable them at your discretion.
